spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -835,7 +843,11 @@ public interface AnnotatedTypeConfigurator<T> {
 * `constructors()` returns a set of `AnnotatedConstructorConfigurator` to configure annotations on constructors.
 * `filterConstructors()` returns a `Stream<AnnotatedConstructorConfigurator>` filtered by applying the provided Predicate on `Constructors()`
 
-`AnnotatedMethodConfigurator` and `AnnotatedConstructorConfigurator` are accessed through `AnnotatedTypeConfigurator`:
+[[annotated_method_configurator]]
+
+==== `AnnotatedMethodConfigurator`
```

Comment:
I hesitated to go that way (to avoid to deep level title). But as we already have a 4 deep level title in  2.8.1, it makes sense. I modify the PR accordingly.